pkgs/applications/networking/remote/citrix-workspace/generic.nix

```diff
@@ -201,7 +201,7 @@ stdenv.mkDerivation rec {
     rm $out/opt/citrix-icaclient/lib/UIDialogLibWebKit.so || true
 
     # We support only Gstreamer 1.0
-    rm $ICAInstDir/util/{gst_aud_{play,read},gst_*0.10,libgstflatstm0.10.so}
+    rm $ICAInstDir/util/{gst_aud_{play,read},gst_*0.10,libgstflatstm0.10.so} || true
```

Comment:
If I'm understanding this correctly, some of these files no longer exist. Can the wildcard instead be altered so these files aren't passed to `rm`?

Reply:
Since this package supports lots of older versions of Citrix (see sources.nix), I'm not sure if those are present in the older packages, hence the `|| true`. I have not tested anything older than the last 2 releases as of this writing. I'm assuming some enterprises require older versions of Citrix to operate, given how slow businesses are to adopt newer versions sometimes. If those files are present for older Citrix versions, they may fail to launch. What should we do?

Reply:
Since this package supports lots of older versions of Citrix (see sources.nix), I'm not sure if those are present in the older packages, hence the || true. I have not tested anything older than the last 2 releases as of this writing. I'm assuming some enterprises require older versions of Citrix to operate, given how slow businesses are to adopt newer versions sometimes. If those files are present for older Citrix versions, they may fail to launch. What should we do?

Reply:
My question spawned from my ignorance about how the generic Citrix builder works. This makes sense, and it's fine the way it is.

Reply:
It took me a good while to understand how this package works/builds. Hopefully it can be cleaned up more in the future :)